https://issues.redhat.com/browse/ACM-10866 (#6581)
open-cluster-management/rhacm-docs: clusters/release_notes/whats_new.adoc

- Placements do not select managed clusters that you are deleting. To learn more about placements, see xref:../cluster_lifecycle/placement_overview.adoc#placement-overview[Placement overview].

> - The `discovery-operator` now automatically imports ROSA clusters into {product-title-short} hub clusters. See, link:../discovery/discovery_import.adoc#discovery_import[Import Discovered clusters automatically for ROSA clusters].

Author: ```suggestion
- The `discovery-operator` now automatically imports {rosa} clusters into {product-title-short} hub clusters. See, link:../discovery/discovery_import.adoc#discovery_import[Import Discovered clusters automatically for {rosa} clusters].
```

> - The `discovery-operator` now automatically imports {rosa} clusters into {product-title-short} hub clusters. See, link:../discovery/discovery_import.adoc#discovery_import[Import Discovered clusters automatically for {rosa} clusters].

Reviewer: Your entry would be only about enabling within MCE, and start with something like _You can now enable..._

Pull a little bit from here:

Automatically import supported clusters into your hub cluster with the `discovery-operator` for faster cluster management, without manually importing individual clusters

> - The `discovery-operator` now automatically imports {rosa} clusters into {product-title-short} hub clusters. See, link:../discovery/discovery_import.adoc#discovery_import[Import Discovered clusters automatically for {rosa} clusters].

Author: ```suggestion
- You can now enable a discovered cluster to automatically import supported clusters into your hub cluster with the `discovery-operator`. Import clusters for faster cluster management. See, link:../discovery/enable_discovery_management.adoc#enable-discovered[Enabling a discovered cluster for management].
```

> - You can now enable a discovered cluster to automatically import supported clusters into your hub cluster with the `discovery-operator`. Import clusters for faster cluster management. See, link:../discovery/enable_discovery_management.adoc#enable-discovered[Enabling a discovered cluster for management].

Author: ```suggestion
- You can now enable a discovered cluster to automatically import supported clusters into your hub cluster with the `discovery-operator`. Import clusters for faster cluster management. See, link:../discovery/enable_discovery.adoc#enable-discovered[Enabling a discovered cluster for management].
```
